src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars > :nth-child(-n+4) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  display: flex;
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
+  width: 16px;
```

Comment:
do not repeat all rules, write only that, which is changing everything else will inherit

Reply:
done